docs/contracts/overview/DigitalAssets.md

```diff
@@ -136,6 +136,114 @@ You can filter the `DataChanged` event with the following `dataKey`:
 
 </details>
 
+### Setting metadata for one or multiple `tokenId`s
+
+The function [`setDataBatchForTokenIds(bytes32[],bytes32[],bytes[])`](../../contracts/contracts/LSP8IdentifiableDigitalAsset/LSP8IdentifiableDigitalAsset.md#setdatabatchfortokenids) can be used to set multiple data key-value pairs at once for one or multiple tokenIds.
+
+This function is flexible enough to enable to:
```

Comment:
```suggestion
This function is flexible enough to set one or multiple [data key-value|(../../standards/lsp-background/erc725.md#erc725y-generic-data-keyvalue-store) pairs for:
```

IMO the bullet point list should only feature the differences to improve clarity. Also added a link for the data keys of ERC725Y.